unipop-elastic/src/org/unipop/elastic/document/schema/DocSchema.java

```diff
@@ -23,6 +23,8 @@ default Document toDocument(E element) {
         Object type = fields.getOrDefault("_type", fields.getOrDefault("type", getType(element)));
```

Comment:
Is "type"/"id" still supported? if not they should probably be removed from here..